main.go

```diff
@@ -123,6 +124,18 @@ var Config struct {
 
 	// MaxConcurrentReconciles is the maximum number of concurrent Reconciles which can be run
 	MaxConcurrentReconciles uint `envconfig:"MAX_CONCURRENT_RECONCILES" default:"3"`
+
+	// MetricsReceiverHost is the hostname for connecting to receiver
+	MetricsReceiverHost string `envconfig:"METRICS_RECEIVER_HOST" default:""`
```

Comment:
That should be part of EdgeDevice CR. The operator may oversee devices scattered across different namespaces, belonging to different users and users might have their private Thanos instances.

Reply:
ok. that makes me think. should we deploy Thanos at all? perhaps let the users configure whatever server they want on their own? @pkliczewski?
and in any case, a user may use something other than Thanos. in this case, we need the path configurable as well.

Reply:
For ease of development and testing, I'd suggest to add a script to `tools/` for deploying thanos (similar to `deploy_grafana.sh`) so any developer who wishes to work with it can do that without going through the deployment process.

Reply:
@masayag @pkliczewski @jakub-dzon 
thank you all for your input. I think I will try creating a YAML with deployments for querier and receiver. I will add documentation to one of the md files or a new file if necessary. If there will be a need for more than just a YAML then I will add a script. when I get to it I will consult you regarding the location of new files.